kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/MctsBeneficiaryImportServiceImpl.java

```diff
@@ -160,6 +161,12 @@ public boolean importMotherRecord(Map<String, Object> record) {
         Boolean abortion = (Boolean) record.get(KilkariConstants.ABORTION);
         Boolean stillBirth = (Boolean) record.get(KilkariConstants.STILLBIRTH);
         Boolean death = (Boolean) record.get(KilkariConstants.DEATH);
+        LocalDate mctsUpdatedDateNic = (LocalDate) record.get(KilkariConstants.LAST_UPDATE_DATE);
+
+        //validate if it's an updated record compared to one from database
+        if (mctsUpdatedDateNic == null || (mother.getUpdatedDateNic() != null && mother.getUpdatedDateNic().isAfter(mctsUpdatedDateNic))) {
+            return true;
```

Comment:
why is this short circuiting and returning true here?  Don't you want to actually update the mother record if this is true?